rotkehlchen/chain/substrate/manager.py

```diff
@@ -388,7 +388,7 @@ def _get_chain_properties(
         try:
             chain_properties = SubstrateChainProperties(
                 ss58_format=properties['ss58Format'],
-                token=Asset(properties['tokenSymbol']),
+                token=properties['tokenSymbol'],
```

Comment:
Yes, was reading your previous comment and trying to follow and changed it and boom. Should be an Asset, not string and is initialized because those asset share the same symbol as identifier